subsys/net/lib/sntp/Kconfig

```diff
@@ -5,7 +5,7 @@
 #
 
 menuconfig SNTP
-	bool "SNTP Support"
+	bool "SNTP Support [Deprecated]"
```

Comment:
hmmm, we are deprecating this with what? are we deprecating the feature itself or the API? This is confusing.

Reply:
There is a PR adding SNTP support over sockets, so this should not be removed

Reply:
> There is a PR adding SNTP support over sockets, so this should not be removed

I would take it easy. As long as we want to *remove old impl* for real, we can do it, and then reintroduce a new impl. Removing old impl is already shocking enough, so nobody will notice small things like removing config and then adding it again.

Reply:
> The flow:
> ...
> Is just noise we do not want to see or have.

This is based on so many assumptions if not wishful thinking. It's better to do things step by step, instead of accumulating concurrent todo's.